src/app/components/Link/index.jsx

```diff
@@ -4,20 +4,24 @@ import { string, boolean } from 'prop-types';
 import { C_WHITE, GEL_SPACING } from '../../lib/constants/styles';
 
 const StyledLink = styled.a`
+  padding: ${GEL_SPACING} 0 ${GEL_SPACING};
```

Comment:
I'd suggest moving this padding to the context the `Link` is used, to ensure this component can be easily reused in contexts other than the `FooterList` component.

Reply:
This currently needs to be on this component otherwise the click areas do not match the design specs. In the future if this component is to be re-used outside of FooterLink we could refactor the FooterLink to use Faux block link (which would maintain the click areas). However that would require the dev team discussing Faux block link as a agreed solution and therefore I believe it is pre-optimisation of the Link component

Reply:
OK, we can discuss this separately to this PR.